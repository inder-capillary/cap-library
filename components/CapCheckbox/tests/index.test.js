// import React from 'react';
// import { shallow } from 'enzyme';

// import CapCheckbox from '../index';

describe('<CapCheckbox />', () => {
  it('Expect to have unit tests specified', () => {
    expect(true).toEqual(false);
  });
});
