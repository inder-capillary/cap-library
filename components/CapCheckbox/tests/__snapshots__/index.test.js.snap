// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CapCheckbox /> CapCheckbox renders correctly 1`] = `
<div
  className="cap-checkbox"
>
  <label
    className="ant-checkbox-wrapper"
  >
    <span
      className="ant-checkbox"
      style={Object {}}
    >
      <input
        checked={false}
        className="ant-checkbox-input"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        type="checkbox"
      />
      <span
        className="ant-checkbox-inner"
      />
    </span>
  </label>
</div>
`;
